README.md

```diff
@@ -70,6 +70,8 @@ Selecting the appropriate tag helps maintain organization in the table and provi
 
 ### Adding new multicodecs to the table
 
+#### Guidelines for submitters
```

Comment:
The guidelines here don't include anything about adding new categories. I figured that happens much less frequently and we could look at that once we have something we're happy with here.